assets/app/styles/_components.less

```diff
@@ -169,6 +169,10 @@
     border-radius: @border-radius-md;
     max-width: 600px;
     padding: 5px 10px;
+    // so that .word-break() on children works
+    [flex] {
+      min-width: 0;
```

Comment:
@sg00dwin please take a look

I'm OK with this, but it does potentially impact any place we use the flex attribute. I didn't see any problems in my testing.

/cc @jwforres